### PR TITLE
Add mysql_query script

### PR DESCRIPTION
**Pull request by v0devil**

### CODE REVIEW

File: src/mysql_query.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+#
+# igcollect - PostgreSQL query results
```
Comment (hasegeli) [inline src/mysql_query.py]: "MySQL"

File: src/mysql_query.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#
+# This script executes a single query and prints the results.  The columns
+# returned by the query are going to be appended to the given prefix.
+# The query must return numeric values.
+#
+# Copyright (c) 2016, InnoGames GmbH
```
Comment (hasegeli) [inline src/mysql_query.py]: "2017"

File: src/mysql_query.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+#
+from __future__ import print_function
+
+import MySQLdb
```
Comment (hasegeli) [inline src/mysql_query.py]: Please use `mysql.connector` instead from Oracle.  MySQLdb is not available for Python 3, and I don't think gets much maintenance.

File: src/mysql_query.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+from argparse import ArgumentParser
+from time import time
+
+
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument('--prefix', default='mysql_query')
+    parser.add_argument('--dbname', default='mysql')
```
Comment (hasegeli) [inline src/mysql_query.py]: `default='test'` might me better.  I don't think MySQL has a database called `mysql` by default.

File: src/mysql_query.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    parser.add_argument(
+        '--query',
+        required=True,
+        action='append',
+        dest='queries', )
```
Comment (hasegeli) [inline src/mysql_query.py]: I would prefer to have `)` on a new line same level as `(`.

File: src/mysql_query.py [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+def main():
+    """The main program"""
+    args = parse_args()
+    items = []
+    now = str(int(time()))
+
+    db = MySQLdb.connect(
+        user='root',
+        host="localhost",
```
Comment (hasegeli) [inline src/mysql_query.py]: These 2 should also be arguments.

File: src/mysql_query.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+
+def get_column_data(rows):
+    if len(rows) > 1:
+        raise Exception('Multiple rows')
+    for row in rows:
+        return row.items()
+
+
+def get_row_data(key_column, rows):
```
Comment (hasegeli) [inline src/mysql_query.py]: I would put `rows` as the first argument to make the function signature compatible with the one above.

File: src/mysql_query.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    first_row = rows[0]
+    if key_column not in first_row:
+        raise Exception('Key column is not there')
+    if len(first_row) == 1:
```
Comment (hasegeli) [inline src/mysql_query.py]: I am not sure whether it can happen, but `< 2` would be a safer check.

File: src/mysql_query.py [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+        user='root',
+        host="localhost",
+        db=args.dbname,
+        read_default_file='/etc/mysql/my.cnf', )
+    cur = db.cursor(MySQLdb.cursors.DictCursor)
+    for query in args.queries:
+        cur.execute(query)
+        if not cur.rowcount:
+            raise Exception('No result')
+        else:
```
Comment (hasegeli) [inline src/mysql_query.py]: Unnecessary `else`

File: src/mysql_query.py (hunk 4 above)
Comment (elmatze79) [reply src/mysql_query.py]: the default databases are:
mysql
performance_schema
information_schema
It is highly advised to remove the test database after install

**Pull request merged**